clientupdate/distsign/distsign_test.go

```diff
@@ -285,13 +285,20 @@ type rootKeyPair struct {
 }
 
 func newRootKeyPair(t *testing.T) rootKeyPair {
-	kp := newKeyPair(t)
-	priv, err := parsePrivateKey(kp.privRaw)
+	privRaw, pubRaw, err := GenerateRootKey()
+	if err != nil {
+		t.Fatalf("GenerateRootKey: %v", err)
+	}
+	kp := keyPair{
+		privRaw: privRaw,
+		pubRaw:  pubRaw,
+	}
+	priv, err := parsePrivateKey(kp.privRaw, pemTypeRootPrivate)
```

Comment:
Might be good to write some negative test cases for parsing roots as signing keys and vice versa, and parsing keys with invalid type tags like meaningless strings, near-miss typos, et c.